src/plugins/offline_editing/offline_editing_plugin_gui.cpp

```diff
@@ -30,33 +30,67 @@
 #include <QFileDialog>
 #include <QMessageBox>
 
-
 QgsSelectLayerTreeModel::QgsSelectLayerTreeModel( QgsLayerTree *rootNode, QObject *parent )
   : QgsLayerTreeModel( rootNode, parent )
 {
   setFlag( QgsLayerTreeModel::ShowLegend, false );
   setFlag( QgsLayerTreeModel::AllowNodeChangeVisibility, true );
 }
 
+int QgsSelectLayerTreeModel::columnCount( const QModelIndex &parent ) const
+{
+  Q_UNUSED( parent );
```

Comment:
I'll remove this ^, that is now used, I started to `return 2` but it's more resilient by adding 1 to the parent